.github/CONTRIBUTING.md

```diff
@@ -130,6 +130,13 @@ To link your Codecademy user profile to GitHub:
 - Typos/bugs
 - Plagiarism
 
+## What is the policy for using generative AI?
+
+We will not accept entries that were exclusively generated through an AI tool. We have this policy because:
```

Comment:
I would add a line break after this, like so:
```suggestion
We will not accept entries that were exclusively generated through an AI tool. We have this policy because:

```